instrumentation/http/src/main/java/brave/http/HttpSampler.java

```diff
@@ -75,9 +75,9 @@
     if (request == null) return null;
     HttpAdapter<Object, Void> adapter;
     if (request instanceof HttpClientRequest) {
-      adapter = new HttpClientRequest.ToHttpAdapter((HttpClientRequest) request);
+      adapter = new HttpClientAdapters.ToRequestAdapter((HttpClientRequest) request);
```

Comment:
Should L35 be updates from:

```
 * @deprecated Since 5.8, use {@code RequestSampler<HttpRequest>}.
```

to 

```
 * @deprecated Since 5.8, use {@code SamplerFunction<HttpRequest>}.
```

Reply:
yep!